testing/sample-apps/README.md

```diff
@@ -10,10 +10,16 @@ Ensure that none of the repositories are currently using the image about to be u
 To update the image, first push the update to a backup image (or generate a new one), then switch the address on the three repositories to the backup image one by one. Once all three repositories are pointing to
 the backup image, push the update to the main image and revert the addresses on the repositories back to the original. Be careful to ensure the image names are appropriately stored in secrets.
 
+### Setting up the environment:
+1. Run the `patch.sh` script under `.github/scripts/` folder. You should have a new folder called `opentelemetry-java-instrumentation`
+2. Cd to the new folder, then run `gradle publishToMavenLocal`
+3. Run `rm -rf opentelemetry-java-instrumentation` to delete the folder.
+
 ### Steps to update image:
 1. Use `ada` commands to autheticate into the testing account
-2. Login to ECR Repository: `aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin {REPOSITORY}`
+2. Login to ECR Repository: `aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin {REPOSITORY}`. Create a new repository first if the repository doesn't exist.
 3. Change repository name in the `build.gradle.kts` file under `testing/sample-apps/springboot` or `testing/sample-apps/sprintboot-remote-service`
+4. Change the `tasks.named("jib").enabled` value on the `build.gradle.kts` file from false to true
```

Comment:
Why is it set to false in the first place?

Reply:
Looks like it was changed a while back: https://github.com/aws-observability/aws-otel-java-instrumentation/commit/862695ea62ecc3ee04bc8e6969670151d57019f3

I'm assuming the sample app gradle file was interfering when the ADOT team tries to build their images.